integration_tests/test_str_to_int.py

```diff
@@ -20,4 +20,10 @@ def f():
     i = i32(int(s))
     assert i == -1234
 
+    assert i32(int("  3   ")) == 3
+    assert i32(int("+3")) == 3
+    assert i32(int("\n3")) == 3
+    assert i32(int("3\n")) == 3
+    assert i32(int("\r\t\n3\r\t\n")) == 3
```

Comment:
There is an `AssertionError` among these tests.
Check all the cases if it gives the correct output.

Reply:
We have the error at the following place.
```bash
(lp) lpython$ git diff                                    
diff --git a/integration_tests/test_str_to_int.py b/integration_tests/test_str_to_int.py
index 765dd1e0f..b67dcab32 100644
--- a/integration_tests/test_str_to_int.py
+++ b/integration_tests/test_str_to_int.py
@@ -3,21 +3,27 @@ from ltypes import i32
 def f():
     i: i32
     i = i32(int("314"))
+    print(i)
     assert i == 314
     i = i32(int("-314"))
     i = i32(int("-314"))
+    print(i)
     assert i == -314
     s: str
     s = "123"
     i = i32(int(s))
+    print(i)
     assert i == 123
     s = "-123"
     i = i32(int(s))
+    print(i)
     assert i == -123
     s = "    1234"
     i = i32(int(s))
+    print(i)
     assert i == 1234
     s = "    -1234   "
     i = i32(int(s))
+    print(i)
     assert i == -1234
 
     assert i32(int("  3   ")) == 3
@@ -25,5 +31,6 @@ def f():
     assert i32(int("\n3")) == 3
     assert i32(int("3\n")) == 3
     assert i32(int("\r\t\n3\r\t\n")) == 3
+    print("Done")
 
 f()
(lp) lpython$ lpython integration_tests/test_str_to_int.py
314
314
AssertionError
```
It seems currently `+` and `-` signs are being ignored.